docs/_docs/getting-started.md

```diff
@@ -421,6 +421,117 @@ response.json()
  'layers': []}
 ```
 
+##### Get A Manifest
+
+As shown above, it's fairly simple to get a manifest.
+
+```python
+req = src.NewRequest(
+    "GET",
+    "/v2/<name>/manifests/<reference>",
+    reggie.WithReference("0.1.2dev0"),
+)
+
+response = src.Do(req)
+manifest = resp.json()
+print(manifest)
+
+# The layers, config, and its digest are found here:
+layers = manifest["layers"]
+config_digest = manifest["config"]["digest"]
+```
+
+##### Get A Blob
+
+Let's say that we just retrieved the image config digest via the interaction above,
+and we want to further inspect or tweak it. You might do the following to retrieve
+the config blob:
+
+```python
+import reggie
```

Comment:
not sure import reggie makes much sense in this example since it's not used

Reply:
removed.